rezerwacje-front/src/layouts/Main/components/Topbar/Topbar.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/styles';
+import { AppBar, Toolbar, Badge, Hidden, IconButton } from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import NotificationsIcon from '@material-ui/icons/NotificationsOutlined';
+import InputIcon from '@material-ui/icons/Input';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    boxShadow: 'none'
+  },
+  flexGrow: {
+    flexGrow: 1
+  },
+  signOutButton: {
+    marginLeft: theme.spacing(1)
+  }
+}));
+
+const Topbar = props => {
+  const { className, onSidebarOpen, ...rest } = props;
+
+  const classes = useStyles();
+
+  const [notifications] = useState([]);
+
+  return (
+    <AppBar
+      {...rest}
+      className={clsx(classes.root, className)}
+    >
+      <Toolbar>
+        <RouterLink to="/">
+          <img
+            alt="Logo"
+            src="/img/logo.svg"
+            height="30"
+          />
```

Comment:
Instead of having an address for the logo you can import svg's into jsx as a component
`import Logo from "assets/logo.svg"`